app/api/srch/search.rb

```diff
@@ -115,6 +115,30 @@ class Search < Grape::API
         sresult
       end
 
+      # Request URL should be /api/srch/locations?srchString=QRY[&seq=KEYCOUNT&showCount=NUM_ROWS&pageNum=PAGE_NUM]
```

Comment:
Super! Perhaps we can add this also to `/doc/API.md`?

Reply:
Done